nestkernel/connection_manager.cpp

```diff
@@ -1117,14 +1117,7 @@ nest::ConnectionManager::get_connections(
 
   if ( source == 0 and target == 0 )
   {
-#ifdef _OPENMP
-#pragma omp parallel
-    {
-      thread t = kernel().vp_manager.get_thread_id();
-#else
-    for ( thread t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
-    {
-#endif
```

Comment:
@jakobj I have been wondering for some time why we ever put that loop into our code: If we do not have OpenMP, then we have no threads, and so everything is by default on thread 0. So it is perfectly fine to drop it.